### PR TITLE
[T02]:Display the first 100 jobs

### PR DESCRIPTION
**Pull request by vinhvinhvinh**
Stories:
Link code feature: https://github.com/fabatek/onboarding-job-board-frontend/tree/feature/vinhnt/t02_display_the_first_100_jobs
Link deploy surge: http://job_board_vinhnt.surge.sh/
Link API to see specific job totals: https://623804ad0a54d2ceab6f951b.mockapi.io/api/jobs

Notes:

1. File modified
- Home/index.tsx
- Home/style.scss

2. Add File/Folder
- components/Main
- add class "card__body" (main.scss)

3. Deploy Update
Result:
image

4. Fix review on Fri
- Add: 
- Loading Screen:
image

### CODE REVIEW

File: src/components/Main/MainContent.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import "./main.scss";
+import React from "react";
+import { useSelector } from "react-redux";
+import { RootStore } from "../../redux/store";
+import { JobType } from "../../redux/jobs/jobs-action-types";
+
+const MainContent: React.FC = () => {
+  const jobState = useSelector((state: RootStore) => state.job);
+  console.log(jobState);
```
Comment (ducdmfaba) [inline src/components/Main/MainContent.tsx:9]: Remove test code

File: src/components/Main/MainContent.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <>
+      <h2>Top Jobs</h2>
+      <div className="d-flex flex-wrap justify-content-around">
+        {jobState.job?.map((item: JobType) => {
+          return (
+            <div key={item.id} className="card">
+              <img
+                width="170"
+                height="170"
```
Comment (ducdmfaba) [inline src/components/Main/MainContent.tsx:19]: Is this work? Look like it is incorrect

File: src/components/Main/main.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    justify-content:space-between;
+    align-items: center;
+    border-radius:0;
+}
+
+.card>img{
+    object-fit:contain;
+}
+
+.card-body{
```
Comment (ducdmfaba) [inline src/components/Main/main.scss:17]: Please follow BEM convention.

File: src/components/Main/MainContent.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                className="card-img-top"
+                alt=""
+              />
+              <div className="card-body">
+                <h5 className="card-title">{item.name}</h5>
+                <p className="card-text">{item.description}</p>
+                <p className="card-text">{item.address1}</p>
```
Comment (ducdmfaba) [inline src/components/Main/MainContent.tsx:27]: Please follow BEM convention.

**Review by ducdmfaba — CHANGES_REQUESTED** [review]

File: src/components/Main/MainContent.tsx (hunk 2 above)
Comment (vinhvinhvinh) [reply src/components/Main/MainContent.tsx:19]: yes this work, i've try to remove this and then 
image

File: src/components/Main/main.scss (hunk 3 above)
Comment (vinhvinhvinh) [reply src/components/Main/main.scss:17]: this 'card-body' is bootstrap's class. Should I add 'card__body' after that ?! Like :
image

File: src/components/Main/MainContent.tsx (hunk 4 above)
Comment (vinhvinhvinh) [reply src/components/Main/MainContent.tsx:27]: same above

**Review by ducdmfaba — APPROVED** [review]

File: src/components/Main/main.scss (hunk 3 above)
Comment (ducdmfaba) [reply src/components/Main/main.scss:17]: Yes. If wanna overwrite the style of bootstrap, we should create new class,

**Comment by ducdmfaba** [comment]
@vinhvinhvinh Please let me know when you have done fixing review on Fri

**Comment by vinhvinhvinh** [comment]
> @vinhvinhvinh Please let me know when you have done fixing review on Fri

**Comment by vinhvinhvinh** [comment]
i have done fixing what i got on Fri, please check Sir. Sorry for my delay

**Pull request merged**